src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -149,7 +149,7 @@ public function pFileWithoutNamespace(FileWithoutNamespace $fileWithoutNamespace
         return ltrim($content);
     }
 
-    protected function p(Node $node, $parentFormatPreserved = false): string
+    protected function p(Node $node, mixed $parentFormatPreserved = false): string
```

Comment:
@TomasVotruba this possibly need to be skipped? the parent `p` method has signature:

```php
     * @param Node $node Node to be pretty printed
     * @param bool $parentFormatPreserved Whether parent node has preserved formatting
     *
     * @return string Pretty printed node
     */
    protected function p(Node $node, $parentFormatPreserved = false) : string {
```

Reply:
I think to reduce unwanted change, the solution is to skip if no `@param mixed` doc https://github.com/rectorphp/rector-src/pull/2701/commits/a569a4b410fbbc9d24527e169f7f76747a16745e